src/app.js

```diff
@@ -0,0 +1,22 @@
+require('dotenv').config;
```

Comment:
Faltou o () no final do config :+1: 
```suggestion
require("dotenv").config();
```